Plan/bukkit/src/main/java/com/djrapitops/plan/Plan.java

```diff
@@ -62,7 +63,18 @@ public class Plan extends JavaPlugin implements PlanPlugin {
 
     @Override
     public void onLoad() {
-        abstractionLayer = new BukkitPlatformLayer(this);
+        if (isJava17OrLater() && isFolia()) {
```

Comment:
```suggestion
        if (isFolia()) {
```
Folia itself requires Java 17+ so this check is redundant